src/equiv.js

```diff
@@ -59,6 +59,10 @@ QUnit.equiv = (function() {
 		return false;
 	}
 
+	function getRegExpFlags( regexp ) {
+		return regexp.flags || regexp.toString().match(/[gimuy]*$/)[0];
```

Comment:
missing a space between arguments list and inside square brackets:

- `(/[gimuy]*$/)` => `( /[gimuy]*$/ )`
- `[0]` => `[ 0 ]`